packages/frontend-2/components/settings/workspaces/members/MembersTable.vue

```diff
@@ -147,11 +153,12 @@ import { usePaginatedQuery } from '~/lib/common/composables/graphql'
 graphql(`
   fragment SettingsWorkspacesMembersTable_WorkspaceCollaborator on WorkspaceCollaborator {
     id
-    projectRoles {
-      project {
-        id
-      }
-    }
+    email
+    # projectRoles {
```

Comment:
we commented out this code during the WS launch, pending better UX, so can be disabled in the query as well